src/main/java/core/basesyntax/service/FruitService.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.FruitStrategy;
+import java.util.List;
+
+public interface FruitService {
+    void getAllTypeStrategy(List<FruitTransaction> fruitTransaction,
```

Comment:
incorrect method name. Does this method return anything? Let's make its purpose more clear. Also, let's pass fruitStrategy to the constructor instead of method parameter